test/e2e/suite/005-app-wide-update-strategy/99-delete.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 120
+delete:
+- apiVersion: v1
+  kind: Namespace
+  name: image-updater-005-01
+- apiVersion: v1
+  kind: Namespace
+  name: image-updater-005-02
```

Comment:
the 2 namespaces created are `image-updater-e2e-005-01` and `image-updater-e2e-005-02`